### PR TITLE
Publish Swift chart

### PR DESCRIPTION
**Pull request by tamalsaha**
Swift is an Ajax proxy for Helm Tiller https://github.com/appscode/swift

### CODE REVIEW

File: stable/swift/templates/_helpers.tpl [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 24 -}}
```
Comment (unguiculus) [inline stable/swift/templates/_helpers.tpl]: Truncate to 63

File: stable/swift/templates/_helpers.tpl [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 45 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+*/}}
+{{- define "fullname" -}}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- printf "%s-%s" $name .Release.Name | trunc 45 -}}
```
Comment (unguiculus) [inline stable/swift/templates/_helpers.tpl]: Truncate to 63

**Review by unguiculus — CHANGES_REQUESTED** [review]
* Please allow for configuring container resources.
* We've started namespacing templates. Please see #1785 and apply accordingly.

**Comment by tamalsaha** [comment]
@unguiculus PTAL

**Comment by tamalsaha** [comment]
It seems circle ci setup has problem. I see no error.

**Comment by unguiculus** [comment]
The error is because your chart does not have any tests. You can ignore this for now but in the near future we will require tests for any new charts.

Please consider my comment about namespacing templates.

**Comment by tamalsaha** [comment]
oh! I missed that part about namespacing templates. I will check. I wish the rules did not change between 2 pr reviews. :(

**Comment by mattfarina** [comment]
@tamalsaha if you rebase off of master that should either fix your circleci check or it will provide an actionable error.

**Comment by unguiculus** [comment]
@tamalsaha Any change you'll get the namespacing of templates done?

**Comment by tamalsaha** [comment]
I looked up what to do today . What I understood was that I need to update the `_helpers.tpl` to make it namespaced. And then update all the other places where those name/fullname templates are referred. is that all the change?

**Comment by tamalsaha** [comment]
@unguiculus, PTAL. templates are namespaced, containers are more configurable.

**Review by unguiculus — APPROVED** [review]

**Pull request merged**